app/controllers/change_requests_controller.rb

```diff
@@ -19,9 +19,9 @@ def create_change_request
       @change_request = ServiceChangeRequest.create(object_id: params[:service_id], resource_id: Service.find(params[:service_id]).resource_id)
     elsif params[:address_id]
       @change_request = AddressChangeRequest.create(object_id: params[:address_id], resource_id: Address.find(params[:address_id]).resource_id)
-    elsif params[:phone_id] || params[:type] == "phones"
-      resource_id = params[:parent_resource_id] || Phone.find(params[:phone_id]).resource_id
-      @change_request = PhoneChangeRequest.create(object_id: params[:phone_id], resource_id: resource_id)
+    elsif params[:phone_id]
+        resource_id = Phone.find(params[:phone_id]).resource_id
+        @change_request = PhoneChangeRequest.create(object_id: params[:phone_id], resource_id: resource_id)
```

Comment:
Nit: These two lines should be dedented by one level.

```suggestion
      resource_id = Phone.find(params[:phone_id]).resource_id
      @change_request = PhoneChangeRequest.create(object_id: params[:phone_id], resource_id: resource_id)
```